Traceback (most recent call last):
  File "CleanerRunScript/run_bigdansing/run_bigdansing_base.py", line 152, in <module>
    main()
  File "CleanerRunScript/run_bigdansing/run_bigdansing_base.py", line 71, in main
    missing_value_representation='empty'
  File "/home/qzk/MDCBaseline/CleanerRunScript/run_bigdansing/../../util/insert_null.py", line 21, in inject_missing_values
    df = pd.read_csv(csv_file)
  File "/home/qzk/MDCBaseline/baseline_env/lib/python3.6/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/qzk/MDCBaseline/baseline_env/lib/python3.6/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/qzk/MDCBaseline/baseline_env/lib/python3.6/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/qzk/MDCBaseline/baseline_env/lib/python3.6/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/qzk/MDCBaseline/baseline_env/lib/python3.6/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b'Data/5_tax/subset_tax_10k/subset_tax_10k_clean_index.csv' does not exist: b'Data/5_tax/subset_tax_10k/subset_tax_10k_clean_index.csv'
